src/tests/ftest/telemetry/wal_metrics.py

```diff
@@ -0,0 +1,249 @@
+"""
+  (C) Copyright 2018-2024 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+import time
+
+from ior_utils import write_data
+from telemetry_test_base import TestWithTelemetry
+from test_utils_pool import add_pool
+
+
+class WalMetrics(TestWithTelemetry):
+    """Tests for new specific metrics to track activity of md_on_ssd.
+
+    :avocado: recursive
+    """
+
+    def test_wal_commit_metrics(self):
+        """JIRA ID: DAOS-11626.
+
+        The WAL commit metrics is per-engine metrics in 'dmabuff', it includes 'wal_sz', 'wal_qd'
+        and 'wal_waiters' today (see dma_metrics_init() in src/bio/bio_buffer.c). WAL commit metrics
+        are updated on each local transaction (for example, transaction for a update request, etc.)
```

Comment:
The WAL is actually per-pool, we'd make the metrics per-pool (same to WAL replay & checkpoint metrics). I'll make the change in DAOS-15358